exercise/curry.js

```diff
@@ -0,0 +1,59 @@
+/*
+## Curry
+
+### Instruction
+
+Create a function `curry` that receives as parameters a string and a
+function.
+
+This function will call the function in parameter with the string inside the parameter of
+the function called.
+
+Create two functions which can only take one argument each.
+Those functions will be passed in the `curry` function:
+
+- toUpper (Only one arg) :
```

Comment:
`toUpper` and `addSpace` are the one that actually curry.

This exercise is mixing callbacks and curry which is a good idea but maybe we need a "just curry" before that so that they understand `a => b => a + b`

I would like to see some of the first exercise from data revisited with curry.

Then we could do an exercise that mix writing curry functions and using them